shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/ConcreteCryptoTypes.hs

```diff
@@ -42,206 +46,210 @@ import qualified Shelley.Spec.Ledger.Tx as Tx
 import qualified Shelley.Spec.Ledger.TxData as TxData
 import qualified Shelley.Spec.Ledger.UTxO as UTxO
 import Test.Cardano.Crypto.VRF.Fake (FakeVRF)
+import qualified Cardano.Crypto.VRF.Class as VRF
+import qualified Cardano.Crypto.DSIGN.Class as DSIGN
+import qualified Cardano.Crypto.Hash as Hash
+import qualified Cardano.Crypto.KES.Class as KES
 
+
+type C = ConcreteCrypto ShortHash
 data ConcreteCrypto (h :: *)
 
+
+type Mock c =
+  ( Crypto c
+  , Num (DSIGN.SignKeyDSIGN (DSIGN c))
+  , Num (VerKeyDSIGN (DSIGN c))
+  , (VRF c) ~ FakeVRF
+  , KES.Signable (KES c) ~ SignableRepresentation
+  , DSIGN.Signable (DSIGN c) ~ SignableRepresentation
+  , HASH c ~ ADDRHASH c
+  )
+
 instance HashAlgorithm h => Crypto (ConcreteCrypto h) where
   type HASH (ConcreteCrypto h) = h
   type ADDRHASH (ConcreteCrypto h) = h
   type DSIGN (ConcreteCrypto h) = MockDSIGN
   type KES (ConcreteCrypto h) = MockKES 10
   type VRF (ConcreteCrypto h) = FakeVRF
 
-type DCert h = Delegation.Certificates.DCert (ConcreteCrypto h)
+type DCert h = Delegation.Certificates.DCert (h)
```

Comment:
This looks like a search/replace, it would be nice to remove the redundant parentheses. Moreover, `h` (hash) should now be `c` (crypto), right?

Reply:
I'm going to delete all of these type synonyms. I really didn't expect the PR to receive all this attention so soon. I wouldn't have put it here in this state.

Reply:
Ah, good.

I'm a bit confused, @JaredCorduan already approved, but the type synonyms are still there :confused:

Reply:
We are addressing a lot of technical debt at the moment, and since this PR touches many files, I don't mind merging now, and removing the type synonyms in a follow up PR. I think the synonyms bother @redxaxder just a much as they bother the rest of us, but it's the path to eradication of `Concrete crypto`!